database/ticket.go

```diff
@@ -91,6 +91,25 @@ type Ticket struct {
 	Outcome TicketOutcome
 }
 
+type TicketList []Ticket
```

Comment:
This is nothing technically incorrect about this, but I'm curious if this is a precursor to something larger?

I ask because I've personally never liked it when code does this in general without a specific need since it hides the type for no real reason.  For example, when I'm looking at the code and am not already familiar with it and I see:

```Go
func (vdb *VspDatabase) GetUnconfirmedTickets() (TicketList, error) {
```

I think, hmm, ok, now I have to go figure out what `TicketList` is.  Is it a map?  A slice?  A generic container? Something else?

In other words, I generally find that it adds more cognitive overhead in general.

Reply:
Interesting point I hadn't considered before. I do anticipate adding more funcs to TicketList later, but nothing imminent. 

It's mainly just to make the code calling `EarliestPurchaseHeight` cleaner, but also has the side-effect of making autocomplete more useful. Probably a habit from my (mostly) OOP background.

Reply:
Gotcha.  Like I said, nothing wrong with it technically.  I just figured I'd offer my perspective since I was reviewing.